aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -1766,7 +1772,7 @@ def _call_route(self, route: Route, route_arguments: Dict[str, str]) -> Response
                 # If the user has turned on debug mode,
                 # we'll let the original exception propagate, so
                 # they get more information about what went wrong.
-                return ResponseBuilder(
+                return self.response_builder_class(
```

Comment:
same as before - is there a reason why we are allocating extra memory to have `ResponseBuilder` into a public attribute?